Client/Frontend/Share/ShareExtensionHelper.swift

```diff
@@ -13,8 +13,7 @@ class ShareExtensionHelper: NSObject {
 
     fileprivate let selectedURL: URL
     fileprivate var onePasswordExtensionItem: NSExtensionItem!
-    // Wechat share extension doesn't like our default data ID which is a modified to support password managers.
-    fileprivate let customDataTypeIdentifers = ["com.tencent.xin.sharetimeline"]
+    fileprivate let browserFillIdentifier = "org.appextension.fill-browser-action"
```

Comment:
Do we need a follow up bug filed for "Wechat share extension" ?

Reply:
No the issue with wechat was the same as Notes. Sending `public.url` now to everything but password managers fixes wechat too.